sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -137,6 +138,7 @@ function buildPipelineRequest(
 interface RequestBody {
   body?: RequestBodyType;
   formData?: FormDataMap;
```

Comment:
Do we still need this `formData?: FormDataMap;`?

Reply:
No, I guess we don't need it right now. We may need to re-add this at some point, though, if we ever need to support `application/x-www-form-urlencoded`.